features/yield/FarmingCalculator.tsx

```diff
@@ -133,10 +143,14 @@ const FarmingCalculator = () => {
     setDefaultValues();
 
     if (WEEKLY_UMA_REWARDS[tokenAddress.toLowerCase()]) {
-      // TODO: Display REN rewards as well
-      const _weeklyPoolRewards =
-        WEEKLY_UMA_REWARDS[tokenAddress.toLowerCase()].UMA;
-      setWeeklyPoolRewards(_weeklyPoolRewards.toString());
+      const selectedTokenRewards =
+        WEEKLY_UMA_REWARDS[tokenAddress.toLowerCase()];
+      setWeeklyUmaRewards(selectedTokenRewards.UMA.toString());
+      if (Object.keys(selectedTokenRewards).includes("REN")) {
```

Comment:
Don't love this hard coded solution but it works for now

Reply:
ye, not ideal. we could generalize it but would need to re-build this whole component I think.